sdk/core/azure-core/src/test/java/com/azure/core/util/polling/PollingUtilsTests.java

```diff
@@ -64,4 +81,143 @@ public void missingOrDuplicateSlashInPath() {
         // Case 4: both of endpoint and relative path have slash
         assertEquals(expectedPath, getAbsolutePath(relativePathWithSlash, endpointWithSlash, null));
     }
+
+    @Test
+    public void testConvertResponse() {
+        String status = "Succeeded";
+
+        Resource convertOrigin = new Resource().setName("name").setStatus(status);
+
+        PollResult convertResult = PollingUtils.convertResponseSync(convertOrigin,
+            new DefaultJsonSerializer(),
+            TypeReference.createInstance(PollResult.class));
+
+        assertEquals(status, convertResult.getStatus());
+    }
+
+    @Test
+    public void testSerializeResponse() {
+        String status = "Succeeded";
+
+        Resource convertOrigin = new Resource().setName("name").setStatus(status);
+
+        JsonSerializer customizedJsonSerializer = new JsonSerializer() {
+            private static final ClientLogger LOGGER = new ClientLogger(DefaultJsonSerializer.class);
+
+            private final static SerializerAdapter JACKSON_ADAPTER = new JacksonAdapter((outerMapper, innerMapper) -> {
+                JacksonAnnotationIntrospector annotationIntrospector =
+                    new JacksonAnnotationIntrospector() {
+                        @Override
+                        public JsonProperty.Access findPropertyAccess(Annotated annotated) {
+                            JsonProperty.Access access = super.findPropertyAccess(annotated);
+                            if (access == JsonProperty.Access.WRITE_ONLY) {
+                                return JsonProperty.Access.AUTO;
+                            }
+                            return access;
+                        }
+                    };
+                outerMapper.setAnnotationIntrospector(annotationIntrospector);
+            });
```

Comment:
A prototype of customized JsonSerializer that can serialize `JsonProperty.Access.WRITE_ONLY` property.